web/src/pages/Cases/CaseDetails/Overview/DisputeBasicInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import styled from "styled-components";
+import { StyledSkeleton } from "components/StyledSkeleton";
+import { isUndefined } from "utils/index";
+
+const QuestionAndDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+  > * {
+    margin: 0px;
+  }
+`;
```

Comment:
let's take the margin out and make it more specific using styled components
```suggestion
const QuestionAndDescription = styled.div`
  display: flex;
  flex-direction: column;
`;

const StyledReactMarkDown = styled(ReactMarkDown)`
  margin: 0px;
`
```